README.md

```diff
@@ -5,20 +5,22 @@ Ivy is an interpreter for an APL-like language. It is a plaything and a work in
 progress.
 
 Ivy has a custom domain. Do not install using github directly. Instead, run:
+
 	go get robpike.io/ivy
 
-Documentation at http://godoc.org/robpike.io/ivy.
+Documentation at https://godoc.org/robpike.io/ivy.
+
 Try the demo:
+
 	cd demo; go run demo.go
 
 Prototype apps for iPhone, iPad, and Android are available in the App store and Google Play store.
 To find them, search for "ivy bignum calculator".
 
-Slides for a talk at: http://go-talks.appspot.com/github.com/robpike/ivy/talks/ivy.slide
-Video for the talk at: http://www.youtube.com/watch?v=PXoG0WX0r_E
+Slides for a talk at: https://talks.godoc.org/github.com/robpike/ivy/talks/ivy.slide  
```

Comment:
trailing spaces

Reply:
They are intentional and significant for markdown formatting. They make these lines end with a hard break, instead of being merged into a single paragraph. From https://daringfireball.net/projects/markdown/syntax#p:

> When you do want to insert a \<br /\> break tag using Markdown, you end a line with two or more spaces, then type return.

You can see their effect by comparing the rendered README at https://github.com/shurcooL/ivy/tree/patch-1#readme vs the one in this repo currently.

If you do want this to continue to be a single paragraph, let me know, I can remove them.